pages/your_tasks/gitlab_zenodo.md

```diff
@@ -0,0 +1,78 @@
+3. Increase the discoverability of your software through Zenodo's search functionality and metadata.
+4. Comply with funding agency requirements for open science and data sharing.
+5. Provide a stable, versioned archive of your software for reproducibility purposes.
+
+In this document, we describe how to automatically archive a snapshot of your code repository on Zenodo whenever you create a new release on GitLab. 
+
+### Considerations
+
+* You use a GitLab instance (either on gitlab.com or self-hosted) as your software repository
+* You make software releases (e.g. using [semantic versioning][semantic-versioning])
+* You use or are happy to make an account on the [Zenodo][zenodo] archive
+* You would like to archive your code on Zenodo and obtain a DOI for each software release that people can use to cite your work
+* You can provide metadata for your software in a `.zenodo.json` file or a `codemeta.json` file in the root of your repository to specify the metadata for your Zenodo deposit.
+* Your GitLab repository can run CI/CD pipelines.
+
+### Solutions
+
+* Provide software metadata in a `.zenodo.json` or `codemeta.json` file.
+  * Create a `.zenodo.json` file in the root directory of your repository following the [Zenodo's metadata schema](https://developers.zenodo.org/#metadata).
+  * Use the [Zenodo's sandbox](https://sandbox.zenodo.org/) to test your metadata.
+  
+* Get a [Zenodo token](https://zenodo.org/account/settings/applications/tokens/new/) with publishing rights (`deposit:actions` and `deposit:write` scopes).
+
+* Add the [eossr](https://escape-ossr.gitlab.io/eossr/gitlab_to_zenodo.html) or the [gitlab2zenodo](https://gitlab.com/sbeniamine/gitlab2zenodo) libraries in  your [GitLab-CI pipeline](https://docs.gitlab.com/ee/ci/yaml/) to automatically archive your code on Zenodo whenever you create a new release.
+
+* [Create a GitLab release](https://everse.software/RSQKit/releasing_code) in your repository.
+  * Go to your repository
+  * Click on releases and then on New release
+  * Use the same version number as specified in your metadata (e.g. `1.0.0`).
+  * Add release notes
+  * Click on Create release
+
+* The CI pipeline will automatically push your code on Zenodo.
+
+* Get the DOI:
+   * After the GitLab release, go to Zenodo, where you will see your repository archived with a DOI assigned to it. Your badge is available in "Details":
+
+![Badge in Zenodo](../../images/badge_zenodo.png)
+
+* You may copy the badge in your README file:
+   * Now you can add the Zenodo badge in your GitLab repository. Click on the blue DOI and copy the markdown in your README. It will show as follows:
+
+![Badge in your repository](../../images/badge_in_repo.png)
+
+## Tools and resources 
+
+* Tools: [GitLab][gitlab], [Zenodo][zenodo], [eossr](https://escape-ossr.gitlab.io/eossr/), [gitlab2zenodo](https://gitlab.com/sbeniamine/gitlab2zenodo)
```

Comment:
This one, please leave blank (remove header)
We will link to the tools in our `yaml` file, and this section will be auto-populated.

We are changing the task template to reflect this.